SCClassLibrary/JITLib/ProxySpace/NodeProxy.sc

```diff
@@ -16,6 +16,7 @@ NodeProxy : BusPlug {
 
 	init {
 		nodeMap = ProxyNodeMap.new;
+		nodeMap.put(\fadeTime, 0.02);
```

Comment:
as detailed in the above discussion, this would be better written as a `classvar <defaultFadeTime` and not a magic number that's duplicated in 3 places

Reply:
Oh yes, sorry I missed that. Really happy to see this magic number gone.